README.md

```diff
@@ -1 +1,52 @@
-# drone-httprequest
\ No newline at end of file
+A plugin to http request plugin for Drone.
+
+# Usage
+
+The following settings changes this plugin's behavior.
+
+* param1 (optional) does something.
+* param2 (optional) does something different.
+
+Below is an example `.drone.yml` that uses this plugin.
+
+```yaml
+kind: pipeline
+name: default
+
+steps:
+- name: run senthilhns/drone_http_request_plugin plugin
+  image: senthilhns/drone_http_request_plugin
+  pull: if-not-exists
+  settings:
+    param1: foo
+    param2: bar
+```
+
+# Building
+
+Build the plugin binary:
+
+```text
+scripts/build.sh
+```
+
+Build the plugin image:
+
+```text
+docker build -t senthilhns/drone_http_request_plugin -f docker/Dockerfile .
+```
+
+# Testing
+
+Execute the plugin from your current working directory:
+
+```text
+docker run --rm -e PLUGIN_PARAM1=foo -e PLUGIN_PARAM2=bar \
+  -e DRONE_COMMIT_SHA=8f51ad7884c5eb69c11d260a31da7a745e6b78e2 \
+  -e DRONE_COMMIT_BRANCH=master \
+  -e DRONE_BUILD_NUMBER=43 \
+  -e DRONE_BUILD_STATUS=success \
+  -w /drone/src \
+  -v $(pwd):/drone/src \
+  senthilhns/drone_http_request_plugin
```

Comment:
update the example with inputs supported by httpRequest plugin

Reply:
updated